core/parser.mly

```diff
@@ -413,6 +430,7 @@ arg:
 | UFLOAT                                                       { string_of_float' $1 }
 | TRUE                                                         { "true"  }
 | FALSE                                                        { "false" }
+| DEFAULT                                                      { "default" }
```

Comment:
this was missing, making impossible to reset settings to default since
```ocaml
links> @set something default ;
```
caused a parsing error

Reply:
You should submit this in a separate PR, do not include it in this one.

Reply:
ok, i ll do that